core/storage/user/gae_models.py

```diff
@@ -945,7 +948,8 @@ def get_model_association_to_user():
     def get_export_policy(cls):
         """Model contains data to export corresponding to a user."""
         return dict(super(cls, cls).get_export_policy(), **{
-            'activity_ids': base_models.EXPORT_POLICY.EXPORTED,
+            'activity_ids': base_models.EXPORT_POLICY.NOT_EXPORTED,
```

Comment:
@vojtechjelinek, Changing this from `EXPORTED` to `NOT_EXPORTED` gives the following error: 
AttributeError: type object 'Enum' has no attribute 'NOT_EXPORTED'

Reply:
Oh, sorry it should be `NOT_APPLICABLE`

Reply:
Done.